cfn-resources/maintenance-window/cmd/resource/resource.go

```diff
@@ -0,0 +1,215 @@
+package resource
+
+import (
+	"context"
+	"errors"
+
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
```

Comment:
```suggestion
	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
```

Reply:
i had to revert this Andrea, it goes against a lit rule (the import order and distribution is done automatically by the go imports)